assets/css/style.css

```diff
@@ -0,0 +1,86 @@
+h1 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #227452;
+}
+
+p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  margin: 2em 24px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.headline {
+  background-color: #fff;
+  background-image: url("../../assets/img/background-image-mobile.svg");
```

Comment:
- [ ] Please 🙏🏽 correct the position of the background image to look similar to the Figma template.

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/212431053-0e711265-9a6c-4997-b1b8-81c893411b8f.png) | ![image](https://user-images.githubusercontent.com/58642949/212431152-9592f71d-2caf-4090-9a69-b07c78074784.png) |

Reply:
Hello @franclobo, thanks for the help, I think I just need to fix the background, could you guide me a bit please? Thank you!